src/Domain/LeanCode.DomainModels.MassTransitRelay/Middleware/CommitDatabaseTransactionMiddleware.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+
+namespace LeanCode.DomainModels.MassTransitRelay.Middleware;
+
+public class CommitDatabaseTransactionMiddleware<TDbContext>
```

Comment:
Now that it has been split away from `PublishEvents`, this could be put in some separate package, closer to some generic EF stuff. I can imagine this being useful in a vacuum, even without MT (and even though we don't have backends without MT nowadays).

Reply:
I don't think it's worth it now...

Reply:
I also think it is not worth it. It is usable separately, but the idea is that it is part of the relay/outbox integration.